Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM python:3-slim
+
+LABEL maintainer="Penn Labs"
+
+# Install build dependencies
+RUN apt-get update && apt-get install --no-install-recommends -y default-libmysqlclient-dev gcc \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install pipenv
+RUN pip install pipenv
+
+WORKDIR /app/
+
+# Copy project dependencies
+COPY Pipfile* /app/
+
+# Install project dependencies
+RUN pipenv install --system
+
+# Copy project files
+COPY . /app/
+
+
+# Run uWSGI
+CMD ["/usr/local/bin/uwsgi", "--ini", "/app/setup.cfg"]
```

Comment:
does this docker image serve anything static, and if so, do we need mime types?

Reply:
I don't think there's anything static being served